### PR TITLE
Feature/docker deployment

### PR DESCRIPTION
**Pull request by Shahana2601**
Created containers for deploying code into docker enigne, added files including Dockerfile, docker-compose.yml and dependencies.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -24,6 +25,14 @@ repositories {
 	maven { url 'https://repo.spring.io/snapshot' }
 }
 
+docker {
```
Comment (ashutosh-2609) [inline build.gradle]: I feel you can just modify your build.gradle to generate jar file instead line 28 to 35 ??

**Review by ashutosh-2609 — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 1 above)
Comment (Shahana2601) [reply build.gradle]: I have updated the code

File: build.gradle [hunk 2]
```diff
@@ -24,6 +25,14 @@ repositories {
 	maven { url 'https://repo.spring.io/snapshot' }
 }
 
+jar{
```
Comment (ashutosh-2609) [inline build.gradle]: indentation is too much from line 28 to 34 
please reformat it

**Review by ashutosh-2609 — CHANGES_REQUESTED** [review]

File: build.gradle
Comment (divya-sathyan) [inline build.gradle:1]: Refactor the code

File: build.gradle (hunk 2 above)
Comment (Shahana2601) [reply build.gradle]: reformatted

File: build.gradle
Comment (Shahana2601) [reply build.gradle:1]: reformatted

**Review by ashutosh-2609 — APPROVED** [review]

File: src/main/java/com/theelixrlabs/healthcare/dao/DoctorPatientAssignmentRepository.java
Comment (TruptiPatil015) [inline src/main/java/com/theelixrlabs/healthcare/dao/DoctorPatientAssignmentRepository.java:1]: Why this empty file is being pushed?
any specific reason?

Comment (Shahana2601) [reply src/main/java/com/theelixrlabs/healthcare/dao/DoctorPatientAssignmentRepository.java:1]: The file is deleted during previous PR merge. Accidently added again.

**Review by TruptiPatil015 — APPROVED** [review]

**Review by divya-sathyan — APPROVED** [review]

**Pull request merged**